netbench/netbench/src/units/duration.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+pub use core::time::Duration;
+
+pub trait DurationExt {
+    fn millis(&self) -> Duration;
+    fn seconds(&self) -> Duration {
+        self.millis() * 1000
+    }
+    fn minutes(&self) -> Duration {
```

Comment:
can you add some small tests for these and `ByteExt`

Reply:
yep